.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false
+dist: trusty
```

Comment:
Trusty is quite old. Aren't there newer dist's available at travis?

Reply:
I think it's the [most recent they provide](https://docs.travis-ci.com/user/reference/overview/). I think (but unsure) another solution would be to run the static-test in a docker container.